MAINTENANCE.md

```diff
@@ -89,6 +89,14 @@ strcspn requires the data must end with a NUL, so if we cannot guarantee that th
 
 https://github.com/r-lib/vroom/issues/357 tracks this issue
 
+## Ideas we are not pursuing
+
+The following projects have merit, but would require more maintainer bandwidth than is currently available.
+It is helpful to us to (a) record them explicitly for some possible future and (b) get them out of our open issues.
+
+* We are not currently planning to replicate readr behavior in vroom
+  * https://github.com/r-lib/vroom/issues/186
+
```

Comment:
A list like this? And then if other issues fall in the same category they'll go below that bullet?

Reply:
No, a bullet list of specific issues closed.